addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -13,17 +13,50 @@
 
     @include print-variable('priority-palette-prefix', $o-palette-priority-prefix);
 
-    // Need the colors and name of each color palette
-    $palette-names: ();
+    // Need the colors, name, menu background color and footer background color of each color palette
+    $o-palette-names: ();
+    $-default-cc: ();
+    $-bg-attrs: 'menu', 'footer';
+    @each $attr in $-bg-attrs {
+        // Since the variables built here are currently only used for theme previews we
+        // decided to keep things simple and we deliberately ignore potential overrides
+        // from #{$-bg-attrs}-custom and #{$-bg-attrs}-gradient.
+        $-default-cc: map-merge($-default-cc, ($attr: o-safe-get($o-base-color-palette, $attr, 1)))
+    }
+
     @each $palette-name, $palette in $o-color-palettes {
         @if $palette-name != 'user-palette' {
             @each $key, $value in $palette {
                 @include print-variable('o-palette-#{$palette-name}-#{$key}', $value);
             }
-            $palette-names: append($palette-names, $palette-name);
+            $o-palette-names: append($o-palette-names, $palette-name);
+
+            @each $attr-name, $default-cc-idx in $-default-cc {
+                $-cc-idx: $default-cc-idx;
+                @if map-has-key($palette, $attr-name) {
+                    $-cc-idx: map-get($palette, $attr-name);
+                }
+                $-bg: $o-portal-default-body-bg;
```

Comment:
@qsm-odoo not sure if `$o-portal-default-body-bg` it is the value I should use when `xxx: null`. May be I should retrieve `'body'` from `$o-base-color-palette`?

Reply:
Cannot use `transparent`? Otherwise `$body-bg` I guess?

Reply:
If we use transparent, the trick used by @stefanorigano for the theme svg preview doesn't work. So ok for $body-bg if it never has 'transparent' as value. 
![Screenshot from 2021-09-08 11-03-46](https://user-images.githubusercontent.com/49784121/132480318-a162e321-9866-4440-96b8-1974033083e9.png)